src/client_api.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
-import Long from "long";
-import _m0 from "protobufjs/minimal";
+import * as _m0 from "protobufjs/minimal";
+import Long = require("long");
```

Comment:
This is throwing an error cause of the require vs import. Honestly not sure why would protoc generate it this way 🤔 Unfortunately tests won't catch it, cause we're not actually sending any messages to the database, which I guess is something I should look into in the future

Reply:
Huh, weird. I'll manually edit it.